content/case_histories/bookpurnong/index.rst

```diff
@@ -3,22 +3,33 @@
 Bookpurnong
 ===========
 
-- **Authors**: :ref:`dyang`
+**Spatially constrained inversion for quasi 3D modelling of airborne electromagnetic data – an application for environmental assessment in the Lower Murray Region of South Australia**
+
+- **Authors:** Andrea Viezzoli, Esben Auken, and Tim Munday
+- **Editors:** :ref:`dyang`
 
 .. topic :: Prelude
 
     This case history follows the use of the SkyTEM airborne system to characterize hydrogeological settings.
 
 **Special Thanks**
 
-Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ and SkyTEM for permission to reproduce figures and
-adapt text from the source material. This case history is based upon two articles:
+Thanks to the `Society of Exploration Geophysicists <http://seg.org>`__ for permission to reproduce figures and adapt text from:
```

Comment:
This one is not SEG, this is Exploration Geophysics which I believe is maintained by CSIRO. @dougoldenburg: do you remember if we got explicit permission from CSIRO for this?  (Based on our acknowledgement in Mt. Isa, it looks like it)